### PR TITLE
[WIP] Get ready for Xcode 7.0

### PR DESCRIPTION
**Pull request by mrackwitz**
* [x] Add build setting `ENABLE_TESTABILITY`
* [x] Add constant for new `LastSwiftUpdateCheck`
* [x] Bump last known version
* [x] ~~Handle `PRODUCT_BUNDLE_IDENTIFIER`?~~

### CODE REVIEW

**Comment by segiddins** [comment]
Let's get the tests passing so we can merge this?

File: lib/xcodeproj/constants.rb [hunk 1]
```diff
@@ -25,6 +25,10 @@ module Constants
     #
     LAST_UPGRADE_CHECK  = '0640'
```
Comment (segiddins) [inline lib/xcodeproj/constants.rb]: this should be `0700`?

**Comment by segiddins** [comment]
`PRODUCT_BUNDLE_IDENTIFIER` should be in `COMMON_BUILD_SETTINGS` => `:all` ?

**Comment by mrackwitz** [comment]
@segiddins: But `PRODUCT_BUNDLE_IDENTIFIER` contains a custom value related to the target name, doesn't it?

**Comment by segiddins** [comment]
Dammit, yeah, it defaults to `PRODUCT_BUNDLE_IDENTIFIER = "org.name.${PRODUCT_NAME:rfc1034identifier}";`

**Comment by segiddins** [comment]
:+1:

**Comment by kylef** [comment]
:ok_hand:

**Comment by mrackwitz** [comment]
thanks :+1:

**Pull request merged**